src/main/java/teammates/ui/webapi/action/GetFeedbackSessionsAction.java

```diff
@@ -34,15 +52,26 @@ public ActionResult execute() {
         String courseId = getRequestParamValue(Const.ParamsNames.COURSE_ID);
 
         List<FeedbackSessionAttributes> feedbackSessionAttributes;
-        if (courseId == null) {
-            boolean isInRecycleBin = getBooleanRequestParamValue(Const.ParamsNames.IS_IN_RECYCLE_BIN);
 
-            List<InstructorAttributes> instructors = logic.getInstructorsForGoogleId(userInfo.getId(), true);
+        if (courseId == null) {
+            boolean isRequestAsStudent = getRequestParamValue(Const.ParamsNames.IS_IN_RECYCLE_BIN) == null;
 
-            if (isInRecycleBin) {
-                feedbackSessionAttributes = logic.getSoftDeletedFeedbackSessionsListForInstructors(instructors);
+            if (isRequestAsStudent) {
```

Comment:
There are some fields in FeedbackSessionAttribute that students are not supposed to see. See `GetFeedbackSessionAction` to get a idea about what field to hide.